packages/grid/src/vaadin-grid-selection-column.js

```diff
@@ -76,6 +77,17 @@ class GridSelectionColumn extends GridColumn {
         value: false,
       },
 
+      /**
+       * When true, rows can be selected by dragging mouse cursor over selection column.
+       * @attr {boolean} select-rows-by-dragging
+       * @type {boolean}
+       */
+      selectRowsByDragging: {
```

Comment:
Has this API naming been decided somewhere? Would it then be `setSelectRowsByDragging(true)` in the Java API?

Reply:
There was no discussion regarding API naming. Open to suggestions.

Reply:
Suggested API sounds good to me, so I'd suggest to go with that for now. I'll ask in the team if there are different opinions.

Reply:
Another suggestion from Rolf:
- `drag-selection`
- `setDragSelectionEnabled`

I like the `enabled` part of that in the Java API, for toggling a feature using a boolean.

Reply:
Do we have examples of such API naming? At least the `resizable` mentioned above is [`setResizable(true)`](https://vaadin.com/api/platform/24.1.3/com/vaadin/flow/component/grid/Grid.Column.html#setResizable(boolean)) in Java (instead of `setResizeEnabled`).

Reply:
Good point, not really, apart from `Avatar.setTooltipEnabled`.

Reply:
After internal discussion, decided to go with `dragSelect`.